samplers/jaegerremote/sampler_remote_test.go

```diff
@@ -496,6 +497,7 @@ func TestRemotelyControlledSampler_updateRateLimitingOrProbabilisticSampler(t *t
 				),
 			)
 			err := remoteSampler.updateSamplerViaUpdaters(testCase.res)
+			remoteSampler.Close()
```

Comment:
I think it would be more readable to add 

```go
defer remoteSampler.Close()
```

or

```go
t.Cleanup(remoteSampler.Close)
```

before `err := remoteSampler.updateSamplerViaUpdaters(testCase.res)`

Reply:
Agree, fixed!